android/app/src/main/java/org/fossasia/openevent/fragments/AboutFragment.java

```diff
@@ -176,5 +219,100 @@ private void setUpCustomTab(String url) {
 
         customTabsIntent.launchUrl(getActivity(), uri);
     }
+
+    @Override
+    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        super.onCreateOptionsMenu(menu, inflater);
+
+        inflater.inflate(R.menu.menu_home, menu);
+        // Get the SearchView and set the searchable configuration
+        SearchManager searchManager = (SearchManager)getContext(). getSystemService(Context.SEARCH_SERVICE);
+        searchView = (SearchView) menu.findItem(R.id.action_search_home).getActionView();
+        // Assumes current activity is the searchable activity
+        searchView.setSearchableInfo(searchManager.getSearchableInfo(getActivity().getComponentName()));
+        searchView.setIconifiedByDefault(true); // Do not iconify the widget; expand it by default
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        if (sessionsListAdapter == null) {
+            return;
+        }
+        else{
+            try {
+                if (searchText != null) {
```

Comment:
Don't do 

```Java
if(blah == null) {
    return;
} else {
    // Do something
}
```

This is still nesting 

Please do this instead:
```Java
if(blah == null)
     return;

// Do something
```
It automatically is the else condition